_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,349 @@
+---
+id: be4d0c
+name: Inline link is distinguishable
+rule_type: composite
+
+description: |
+  This rule checks that links that are embedded in a paragraph, list or cell have a way to distinguish them from the surrounding text
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_rules:
+  - e7cc2b
+  - 88407d
+  - 36f116
+  - 66e9f0
+  - 548868
+  - 2803b8
+
+acknowledgements:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+  previous authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any HTML element which:
```

Comment:
I must apologise. I've read this 5 times and I still don't know what it means. Is there a way you can rephrase this to make it clearer?

Reply:
Added a note and some explaining remarks. Hope it helps.